### PR TITLE
Fix/v4 improvments

### PR DESCRIPTION
**Pull request by GoOz**
Massive clean up here.

I removed all the `:global` intructions from UI to make it agnostic, as much as possible at least. 
You can use or not CSS-modules on your app but you'll have to `@extend` the  previous global classes, such as `.coz-btn`, and make them global yourself in you app if you want to. 

Besides that there's 2 micro bugfix about buttons and the nav on mobile.

### CODE REVIEW

**Comment by ptbrowne** [comment]
> I removed all the :global intructions from UI to make it agnostic, as much as possible at least.
You can use or not CSS-modules on your app but you'll have to @extend the previous global classes, such as .coz-btn, and make them global yourself in you app if you want to.

I do not understand, I thought that we were getting rid of placeholders because we did not want the app builders to have to write too much css. It seems we are going in the exact opposite direction here. Am I missing something ?

**Comment by ptbrowne** [comment]
Can we split this PR in 2 ? It seems that there are two things going on here, the fixing of nav and buttons and the refactor for the global classes.

**Comment by ptbrowne** [comment]
OK so I was confused. Since there are no more global classes, if you use css modules when you load cozy-ui AND you want to use cozy-ui classes outside cozy-ui components, you will have to use `@extend`. Another alternative is to load your app styles with css modules, without loading cozy-ui with css modules. This means you will be able to do `<MyComponent className='coz-btn' />` but still will be able to have "type" safety for your app classes.

Bottom line : LGTM 👍  ! Next time, I think it is better not to mix concerns in PRs so try to have PRs that are specific to 1 problem :)

**Comment by GoOz** [comment]
@ptbrowne You're totally right, I just had 5 big commits waiting and handled them poorly. It should be a cleaner PR now :) Thanks
